hbs-helpers.js

```diff
@@ -19,16 +19,11 @@ function getStringID (supportedLocales, key, number) {
 
 
 function localizedBreachDataClasses(supportedLocales, dataClasses, args) {
-  const localizedDataClasses = [];
-  if (dataClasses.constructor === Array) {
+    const localizedDataClasses = [];
     dataClasses.forEach(dataClass => {
```

Comment:
I’m confused here. Before we were checking if dataClasses way an array and then doing X. Now we’re just always assuming it’s an Array and calling `.forEach()`. Was there a case/reason where it was either an array vs a plain string?

Reply:
the mSpy breach has a single data class, which blew up the old `localizedBreachDataClasses` logic before we checked that the argument value was an array.

But now this branch adds `formatDataClassesArray` which transforms the `breach.DataClasses` value from HIBP into an array that we can localize, so now it's always an array.